sway/desktop/render.c

```diff
@@ -755,7 +771,7 @@ static void render_titlebar(struct sway_output *output,
 		}
 
 		render_texture(output->wlr_output, output_damage, title_texture,
-			NULL, &texture_box, matrix, con->alpha, 0, false);
+			NULL, &texture_box, matrix, con->alpha, 0, con->saturation, false);
```

Comment:
We'd probably want to keep the titlebar text's saturation at the default `1.0f` instead of the container's saturation